README.md

```diff
@@ -7,10 +7,14 @@ Built with [LangChain](https://github.com/hwchase17/langchain), [GPT4All](https:
 
 # Environment Setup
 In order to set your environment up to run the code here, first install all requirements:
-
-```shell
-pip3 install -r requirements.txt
-```
+- For NVIDIA GPUs (on Linux):
+   ```shell
+   sh ./install_nvidia.sh
```

Comment:
Heya - I'm randomly jumping in to try out the changes in this PR and I noticed that this should be `sh ./install_cuda.sh` or the file name should be changed.

Reply:
Yes, thanks! it was a typo

Reply:
So, guys on CPU shall still be able to run this on CPU right? If you want GPU, you just run this script?

Reply:
I have not officially tested it, but in theory the nvidia script does nothing more than add the environment variable indicating compilation with nvidia. If nvidia is not present on your device, it should still install itself via that script.

In any case, if you set N_GPU_LAYERS to 0 in the .env file, your GPU will not be used (maybe it will, but for the embedder)